ripple/src/main/webapp/posts.js

```diff
@@ -193,4 +193,13 @@ function createPostElement(userName, userBlobKey, businessName, postBlobKey, cap
   `;
   postElement.innerHTML = content;
   return postElement;
+}
+
+// Unit testing exports set up
+
+function subtract(a, b) {
+  return a - b;
+}
+module.exports = { 
```

Comment:
is this also specifically for testing?

Reply:
Each function to be tested needs to be exported via module.exports

Reply:
Ahh interesting, is that Jest specific or is it just how Javascript works?

Reply:
This is how JS works. I implemented this with Mocha / Chai earlier and had to export functions as well.